src/libs/Pusher/pusher.js

```diff
@@ -165,13 +165,15 @@ function bindEventToChannel(channel, eventName, eventCallback = () => {}) {
         // Only call the event callback if we've received the last packet and we don't have any holes in the complete
         // packet.
         if (chunkedEvent.receivedFinal && chunkedEvent.chunks.length === _.keys(chunkedEvent.chunks).length) {
-            eventCallback(JSON.parse(chunkedEvent.chunks.join('')));
+            const combinedData = chunkedEvent.chunks.join('');
+            const finalData = combinedData.substring(1, combinedData.length - 1);
```

Comment:
Why are we doing this?

Reply:
I just updated the PR with a code comment to explain. I hate it, but this is the only way I could leave things in a working state.